src/components/structures/UserMenu.tsx

```diff
@@ -390,16 +385,13 @@ export default class UserMenu extends React.Component<IProps, IState> {
             }
         }
 
-        let homeButton = null;
-        if (this.hasHomePage) {
-            homeButton = (
-                <IconizedContextMenuOption
-                    iconClassName="mx_UserMenu_iconHome"
-                    label={_t("Home")}
-                    onClick={this.onHomeClick}
-                />
-            );
-        }
+        const homeButton = (
+            <IconizedContextMenuOption
+                iconClassName="mx_UserMenu_iconHome"
+                label={_t("Home")}
+                onClick={this.onHomeClick}
+            />
+        );
```

Comment:
We should add some before/after screenshots of what the user menu looks like to the issue description.

Reply:
Done